build.sbt

```diff
@@ -0,0 +1,52 @@
+lazy val scala212 = "2.12.9"
+lazy val scala211 = "2.11.7"
+lazy val supportedScalaVersions = List(scala212, scala211)
+
+ThisBuild / organization := "com.flextrade.jfixture"
+ThisBuild / name         := "jfixture-scala"
+ThisBuild / version      := "1.0.1-SNAPSHOT"
+
+ThisBuild / organizationName := "FlexTrade"
+ThisBuild / organizationHomepage := Some(url("http://www.flextrade.com/"))
+ThisBuild / description  := "JFixture-Scala adds support for scala types and collections to JFixture"
+ThisBuild / homepage     := Some(url("https://github.com/FlexTradeUKLtd/jfixture-scala"))
+
+ThisBuild / startYear    := Some(2008)
+ThisBuild / licenses     += "MIT License" -> url("http://www.opensource.org/licenses/mit-license.php")
+
+ThisBuild / developers := List(
+  Developer(
+    id = "albx79", name = "Alberto Colombo", email = "albx79@gmail.com", url = url("https://github.com/albx79")
```

Comment:
Can you change this to 
<email>opensource@flextrade.com</email>
            <organization>FlexTrade UK Ltd</organization>
            <organizationUrl>http://www.flextrade.com/</organizationUrl>

Reply:
Okay, done